universe/wrappers/__init__.py

```diff
@@ -15,6 +15,7 @@
 from universe.wrappers.vectorize import Vectorize, Unvectorize, WeakUnvectorize
 from universe.wrappers.vision import Vision
 from universe.wrappers.recording import Recording
+from universe.wrappers.monitoring import Monitored
```

Comment:
BTW, would it be confusing to call this wrapper `Monitor`? We've usually tended towards the simplest noun or verb form of the word.